mmdet3d/datasets/pipelines/loading.py

```diff
@@ -434,6 +436,7 @@ def __init__(self,
                  with_mask=False,
                  with_seg=False,
                  poly2mask=True,
+                 seg_3d_labels_dtype=np.int,
```

Comment:
We should not pass np.int, set it through strings, like 'int32', 'int64'. Otherwise the config in other format (json/yaml) will be unapplicable.